tests/bgp/test_bgp_4-byte_as_trans.py

```diff
@@ -0,0 +1,284 @@
+        'tor1': tor1,
+        'tor2': tor2,
+        'dut_asn': dut_asn,
+        'neigh_asn': neigh_asn[tor1],
+        'neigh2_asn': neigh_asn[tor2],
+        'asn_dict':  neigh_asn,
+        'neighbors': tor_neighbors,
+        'namespace': namespace,
+        'dut_ip_v4': dut_ip_v4,
+        'dut_ip_v6': dut_ip_v6,
+        'neigh_ip_v4': neigh_ip_v4,
+        'neigh_ip_v6': neigh_ip_v6,
+        'neigh2_ip_v4': neigh2_ip_v4,
+        'neigh2_ip_v6': neigh2_ip_v6,
+        'peer_group_v4': peer_group_v4,
+        'peer_group_v6': peer_group_v6,
+        'asic_index': asic_index,
+        'dut_bgp_id': dut_bgp_id,
+        'neigh_bgp_id': neigh_bgp_id,
+        'neigh2_bgp_id': neigh2_bgp_id,
+        'dut_ipv4_network': dut_ipv4_network,
+        'dut_ipv6_network': dut_ipv6_network,
+        'neigh_ipv4_network': neigh_ipv4_network,
+        'neigh_ipv6_network': neigh_ipv6_network
+    }
+
+    logger.info("DUT BGP Config: {}".format(duthost.shell("show run bgp", module_ignore_errors=True)['stdout']))
+    logger.info("Neighbor BGP Config: {}".format(nbrhosts[tor1]["host"].shell("show run bgp")['stdout']))
+    logger.info('Setup_info: {}'.format(setup_info))
+
+    yield setup_info
+
+    # restore config to original state
+    config_reload(duthost)
+    config_reload(tor_neighbors[tor1], is_dut=False)
+
+    # verify sessions are established
+    bgp_facts = duthost.bgp_facts(instance_id=asic_index)['ansible_facts']
+    for k, v in bgp_facts['bgp_neighbors'].items():
+        if v['description'].lower() not in skip_hosts:
+            logger.debug(v['description'])
+            assert v['state'] == 'established'
+
+
+def test_4_byte_asn_translation(setup):
+    cmd = 'vtysh -n {} \
+    -c "config" \
+    -c "no router bgp {}" \
+    -c "router bgp {}" \
+    -c "bgp router-id {}" \
+    -c "bgp log-neighbor-changes" \
+    -c "no bgp ebgp-requires-policy" \
+    -c "no bgp default ipv4-unicast" \
+    -c "bgp bestpath as-path multipath-relax" \
+    -c "neighbor {} peer-group" \
+    -c "neighbor {} peer-group" \
+    -c "neighbor {} remote-as {}" \
+    -c "neighbor {} peer-group {}" \
+    -c "neighbor {} description {}" \
+    -c "neighbor {} timers 3 10" \
+    -c "neighbor {} timers connect 10" \
+    -c "neighbor {} remote-as {}" \
+    -c "neighbor {} peer-group {}" \
+    -c "neighbor {} description {}" \
+    -c "neighbor {} timers 3 10" \
+    -c "neighbor {} timers connect 10" \
+    -c "neighbor {} remote-as {}" \
+    -c "neighbor {} peer-group {}" \
+    -c "neighbor {} description {}" \
+    -c "neighbor {} timers 3 10" \
+    -c "neighbor {} timers connect 10" \
+    -c "neighbor {} remote-as {}" \
+    -c "neighbor {} peer-group {}" \
+    -c "neighbor {} description {}" \
+    -c "neighbor {} timers 3 10" \
+    -c "neighbor {} timers connect 10" \
+    -c "address-family ipv4 unicast" \
+    -c "network {}" \
+    -c "neighbor {} soft-reconfiguration inbound" \
+    -c "neighbor {} route-map FROM_BGP_PEER_V4 in" \
```

Comment:
route-map define is missing in the file, what exactly the route-map contain, similar other route map in the files as well.

Reply:
This is a pre-configured route map that is part of the T2 BGP configuration.